### ISSUE TITLE
Field instructions have poor contrast

### ISSUE THREAD

**Issue by MoritzLost** [verbb/hyper#159]
### Describe the bug

The instructions for subfields in a link have poor contrast to the background and fail WCAG 2.2 Level AA:

![Screenshot 2024-05-22 at 16 50 43](https://github.com/verbb/hyper/assets/10146880/8d91a2e4-c144-45c1-9f01-218879c2bb0e)

![Screenshot 2024-05-22 at 16 50 54](https://github.com/verbb/hyper/assets/10146880/a7a0453f-1b94-414a-8e5d-1b250fe1df89)

Craft uses the `--gray-550` variable for instructions, which is `hsl(210, 13%, 40%)`, this one has better contrast, maybe switch to this color?

The same applies to the select field for the link type, the text has poor contrast to the background of the dropdown.

### Steps to reproduce

1. Add instructions to any field or attribute in a link field layout editor.

### Craft CMS version

5.1.4

### Plugin version

2.0.0

**Comment by engram-design** [comment]
I don't exactly _love_ the lack of contrast now between the field label and the instructions. Visually, they look like they now carry the same hierarchal weight. But in the name of accessibility, we can make this change.

For comparison, see the below:

<img width="313" alt="image" src="https://github.com/verbb/hyper/assets/1221575/0c419711-fc1d-4d3e-b9f3-21ee4a1473f4">

Fixed for the next release. To get this early, run:
- `composer require verbb/hyper:"dev-craft-4 as 1.1.30"`
- `composer require verbb/hyper:"dev-craft-5 as 2.0.0"`

**Comment by MoritzLost** [comment]
@engram-design Thanks, that looks much better! I think the bolding distinguishes the label well enough 👍

**Comment by engram-design** [comment]
Fixed in [2.0.1](https://github.com/verbb/hyper/releases/tag/2.0.1)
